packages/models/src/eservice-template/eserviceTemplate.ts

```diff
@@ -46,6 +47,13 @@ export const EServiceTemplateVersion = z.object({
 });
 export type EServiceTemplateVersion = z.infer<typeof EServiceTemplateVersion>;
 
+export const EServiceTemplateRiskAnalysis = RiskAnalysis.and(
+  z.object({ tenantKind: TenantKind })
```

Comment:
Field called `tenantKind` seems a property of risk analisys entity, meanwhile it represent a restriction on allowed tenant for Eservice instantiation. 
I suggest to change with something more clear, for example `allowedTenantKind`, and, I don't know if could be more usefull using a list instead of single to allow a future multiple tenants for this purpose.